dataflowengine/src/main/scala/io/shiftleft/dataflowengine/language/nodemethods/TrackingPointMethods.scala

```diff
@@ -16,13 +15,12 @@ class TrackingPointMethods(val node: nodes.TrackingPointBase) extends AnyVal {
   /**
     * Convert to nearest AST node
     * */
-  def astNode: nodes.AstNode = {
+  def astNode: nodes.AstNode =
```

Comment:
Please stop removing "unnecessary" brackets. This just creates noise and is personal preference.

Reply:
it's just itching me when i see those... ok then :)